server/dfr3-service/src/main/java/edu/illinois/ncsa/incore/service/dfr3/controllers/RestorationController.java

```diff
@@ -153,6 +153,10 @@ public RestorationSet uploadRestorationSet(@ApiParam(value = "json representing
 
         UserInfoUtils.throwExceptionIfIdPresent(restorationSet.getId());
         restorationSet.setCreator(username);
+        if (restorationSet.getRestorationCurves().size() == 0){
```

Comment:
For the future:

Would it be possible to abstract this test into something more reusable across different entities?